module_3/assignment1_5.js

```diff
@@ -0,0 +1,30 @@
+//5. Can you use return instead of break in loops?
+
+let array = [];
+
+for(let i=0;i<5;i++){
+  console.log(i)  
+  array[i] = i;
+  if(i==2)
+  {
+    break;
+  }
+}
+// Output => 0 1 2
+
+console.log("Using Break array is: ",array); // Using Break array is:  [ 0, 1, 2 ]
+
+for(let i=0;i<5;i++){
+  console.log(i)
+  array[i] = i;
+  if(i==2)
+  {
```

Comment:
If there is only 1 statement for any loop, then braces is not required directly return statement you can use 
if (i == 2) return;

Reply:
Done